src/app/static/src/app/store/modelCalibrate/mutations.ts

```diff
@@ -146,7 +146,15 @@ export const mutations: MutationTree<ModelCalibrateState> = {
 
     [ModelCalibrateMutation.ResetIds](state: ModelCalibrateState) {
         stopPolling(state)
-    }
+    },
+
+    [ModelCalibrateMutation.CalibrateResultFetched](state: ModelCalibrateState, action: PayloadWithType<CalibrateResultWithType>) {
```

Comment:
theres another mutation with the same name here that you forgot to delete i think

Reply:
oops, thanks!